docs/how-to-use-gvfs.md

```diff
@@ -42,7 +42,9 @@ the path mapping and convert automatically.
 
 ### Prerequisites
 
-+ A Hadoop environment with HDFS or other Hadoop Compatible File System (HCFS) implementations like S3, GCS, etc. GVFS has been tested against Hadoop 3.3.1. It is recommended to use Hadoop 3.3.1 or later, but it should work with Hadoop 2.x. Please create an [issue](https://www.github.com/apache/gravitino/issues) if you find any compatibility issues.
+ - GVFS has been tested against Hadoop 3.3.1. It is recommended to use Hadoop 3.3.1 or later, but it should work with Hadoop 2.
+  x. Please create an [issue](https://www.github.com/apache/gravitino/issues) if you find any
+  compatibility issues.
 
 ### Configuration
```

Comment:
The configurations listed below appear to be intended for Java GVFS exclusively. 

Should the Python GVFS require a distinct form?

Reply:
Configuration about Python GVFS is just in the chapter below, the whole structure is designed by jiebao

Reply:
You may need to consider this comprehensively.
There may be something in the Python GVFS chapter that also needs to be updated